test/screenshotter/ss_data.yaml

```diff
@@ -170,6 +170,7 @@ Symbols1: |
     \maltese\degree\pounds\$
     \text{\maltese\degree}
 Text: \frac{a}{b}\text{c~ {ab} \ e}+fg
+TextWithMath: \text{for $a < b$ and $ c < d $}.
```

Comment:
Since you've added a screenshot test you'll also have to generate the image files using `make screenshots` and commit those as well.